runtime/polkadot/tests/weights.rs

```diff
@@ -0,0 +1,347 @@
+	let weight = SessionCall::set_keys::<Runtime>(Default::default(), Default::default()).get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_session_purge_keys_is_correct() {
+	// #[weight = 120_000_000
+	// 	+ T::DbWeight::get().reads_writes(2, 1 + T::Keys::key_ids().len() as Weight)]
+	//
+	// Polkadot has five possible session keys, so we default to key_ids.len() = 5
+	let expected_weight = 770_000_000;
+	let weight = SessionCall::purge_keys::<Runtime>().get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_democracy_propose_is_correct() {
+	// #[weight = 5_000_000_000]
+	let expected_weight = 5_000_000_000;
+	let weight = DemocracyCall::propose::<Runtime>(Default::default(), Default::default()).get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_democracy_vote_is_correct() {
+	use democracy::AccountVote;
+	let vote = AccountVote::Standard { vote: Default::default(), balance: Default::default() };
+
+	// #[weight = 200_000_000]
+	let expected_weight = 200_000_000;
+	let weight = DemocracyCall::vote::<Runtime>(Default::default(), vote).get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_democracy_enact_proposal_is_correct() {
+	// #[weight = T::MaximumBlockWeight::get()]
+	let expected_weight = MaximumBlockWeight::get();
+	let weight =
+		DemocracyCall::enact_proposal::<Runtime>(Default::default(), Default::default()).get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_phragment_vote_is_correct() {
+	// #[weight = 100_000_000]
+	let expected_weight = 100_000_000;
+	let weight = PhragmenCall::vote::<Runtime>(Default::default(), Default::default()).get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_phragment_submit_candidacy_is_correct() {
+	// #[weight = 500_000_000]
+	let expected_weight = 500_000_000;
+	let weight = PhragmenCall::submit_candidacy::<Runtime>().get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_phragment_renounce_candidacy_is_correct() {
+	// #[weight = (2_000_000_000, DispatchClass::Operational)]
+	let expected_weight = 2_000_000_000;
+	let weight = PhragmenCall::renounce_candidacy::<Runtime>().get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_treasury_propose_spend_is_correct() {
+	// #[weight = 120_000_000 + T::DbWeight::get().reads_writes(1, 2)]
+	let expected_weight = 345_000_000;
+	let weight =
+		TreasuryCall::propose_spend::<Runtime>(Default::default(), Default::default()).get_dispatch_info().weight;
+
+	assert_eq!(weight, expected_weight);
+}
+
+#[test]
+fn weight_of_treasury_approve_proposal_is_correct() {
+	// #[weight = (34_000_000 + T::DbWeight::get().reads_writes(2, 1), DispatchClass::Operational)]
+	let expected_weight = 184_000_000;
```

Comment:
You should be using the formula here, not a raw number. Note that `T::DbWeight` could change, and I dont think that should necessarily make all these tests fail